test/cpp/qps/report.h

```diff
@@ -103,6 +104,29 @@ class GprLogReporter : public Reporter {
   void ReportTimes(const ScenarioResult& result) GRPC_OVERRIDE;
 };
 
+/** Reporter for performance database tool */
+class PerfDbReporter : public Reporter {
+ public:
+  PerfDbReporter(const string& name, const string& access_token, const string& test_name, 
+    const string& sys_info, const string& server_address, const string& tag)
+   : Reporter(name), access_token_(access_token), test_name_(test_name), sys_info_(sys_info), tag_(tag) {
+    perfDbClient_.init(grpc::CreateChannel(server_address, grpc::InsecureCredentials(), ChannelArguments()));
+  }
+  ~PerfDbReporter() { SendData(); };
```

Comment:
Was unable to add GRPC_OVERRIDE to the constructor, got error "‘grpc::testing::PerfDbReporter::PerfDbReporter(const string&, const string&, const string&, const string&, const string&, const string&)’ marked override, but does not override".